src/coord/src/coord.rs

```diff
@@ -445,7 +435,7 @@ where
 
         // Cleanly drain any pending messages from the worker before shutting
         // down.
-        drop(internal_cmd_tx);
+        drop(self.internal_cmd_tx);
```

Comment:
I'm surprised that I was able to type this. I either don't understand how drop works or this isn't doing what I think it is